src/microbe_stage/Microbe.cs

```diff
@@ -753,8 +773,26 @@ public void PlaySoundEffect(string effect)
         // TODO: make these sound objects only be loaded once
         var sound = GD.Load<AudioStream>(effect);
 
-        otherAudio.Stream = sound;
-        otherAudio.Play();
+        // Find a player not in use or create a new one if none are available.
+        var player = otherAudioPlayers.Find(nextPlayer => !nextPlayer.Playing);
+
+        if (player == null)
+        {
+            // If we hit the player limit just return and ignore the sound.
+            if (otherAudioPlayers.Count >= 10)
```

Comment:
Could you put this constant in Constants, something like MAX_CONCURRENT_SOUNDS_PER_ENTITY